bokeh/plotting_helpers.py

```diff
@@ -469,7 +469,7 @@ def _new_xy_plot(x_range=None, y_range=None, plot_width=None, plot_height=None,
 
 
 def _handle_1d_data_args(args, datasource=None, create_autoindex=True,
-        suggested_names=[]):
+        suggested_names=None):
```

Comment:
Sure, I could do that here. The function is never called with the `suggested_names` arguments, so I suppose I can just remove that bit and be done with it?